srcpkgs/pd/template

```diff
@@ -11,10 +11,10 @@ makedepends="alsa-lib-devel jack-devel tk-devel fftw-devel"
 depends="tk"
 short_desc="The Pure Data real-time music and multimedia environment"
 maintainer="Andrea Brancaleoni <abc@pompel.me>"
-license="BSD"
+license="BSD-3-Clause"
 homepage="http://msp.ucsd.edu/software.html"
 distfiles="http://msp.ucsd.edu/Software/pd-$_ver.src.tar.gz"
-checksum=e89db1ca58ae3d3f5a94f68ddb0ace9cc02c8af28aa19a4ae4f0ddeeb7a24da3
+checksum=681be08494b9f29ba00e01d9f5b2adc6fa989e139071b1cf2034cfff9885065d
 
 pre_configure() {
 	autoreconf -fi
```

Comment:
Use `./autogen.sh` instead.

See the error on Travis: https://travis-ci.org/void-linux/void-packages/jobs/418328922#L849

Reply:
done